plugins/outputs/sql/sql.go

```diff
@@ -58,6 +69,24 @@ func (*SQL) SampleConfig() string {
 	return sampleConfig
 }
 
+func (p *SQL) Init() error {
+	if p.TableExistsTemplate == "" {
+		p.TableExistsTemplate = "SELECT 1 FROM {TABLE} LIMIT 1"
+	}
+	if p.TimestampColumn == "" {
+		p.TimestampColumn = "timestamp"
+	}
+	if p.TableTemplate == "" {
+		if p.Driver == "clickhouse" {
+			p.TableTemplate = "CREATE TABLE {TABLE}({COLUMNS}) ORDER BY ({TAG_COLUMN_NAMES}, {TIMESTAMP_COLUMN_NAME})"
+		} else {
```

Comment:
Do we need a case for unknown/unsupported p.Driver value?
something like:
```
if p.TableTemplate == "" {
		switch p.Driver {
		case "clickhouse":
			p.TableTemplate = "CREATE TABLE {TABLE}({COLUMNS}) ORDER BY ({TAG_COLUMN_NAMES}, {TIMESTAMP_COLUMN_NAME})"
		case "sqlite", "postgres", "mysql": // Add more supported drivers if necessary
			p.TableTemplate = "CREATE TABLE {TABLE}({COLUMNS})"
		default:
			return fmt.Errorf("unsupported driver: %s", p.Driver)
		}
	}
```

Reply:
I think we should keep the template default setting as is but adding a check for the driver is a good thing to have. Added.